src/app/wizard/step/cluster/component.ts

```diff
@@ -366,6 +427,10 @@ export class ClusterStepComponent extends StepBase implements OnInit, ControlVal
     );
   }
 
+  isDualStackIPFamilySelected(): boolean {
```

Comment:
This is not a getter because getters are generally used to return property of the class (although we've used them in other contexts in our code). But here this method is also consistent with how other methods are written in this component.